src/js/entities-service/form-responses.js

```diff
@@ -14,7 +14,11 @@ const Entity = BaseEntity.extend({
     if (!responseId) return [{}];
     return fetcher(`/api/form-responses/${ responseId }/response`).then(handleJSON);
   },
-  fetchLatestSubmission(patientId, formId) {
+  fetchLatestSubmission(patientId, formId, flowId) {
+    if (flowId) {
+      return fetcher(`/api/patients/${ patientId }/form-responses/latest?filter[form]=${ formId }&filter[flow]=${ flowId }`).then(handleJSON);
+    }
```

Comment:
Using the data object:

```js
fetchLatestSubmission(patientId, formId, flowId) {
  const data = {
    filter: {
      form: formId,
    },
  };

  if (flowId) data.filter.flow = flowId;

  return fetcher(`/api/patients/${ patientId }/form-responses/latest`, { data }).then(handleJSON);
}
```

Causes this error saying we can't do that for `GET` requests:

![Screen Shot 2022-12-01 at 2 28 59 PM](https://user-images.githubusercontent.com/35355575/205153108-0f99eca6-7489-4b48-a952-4e8e9d9ad6e9.png)

Reply:
Could just do something like this instead:

```js
fetchLatestSubmission(patientId, formId, flowId) {
  let url = `/api/patients/${ patientId }/form-responses/latest?filter[form]=${ formId }`;

  if (flowId) url += `&filter[flow]=${ flowId }`; // or use concat()

  return fetcher(url).then(handleJSON);
}
```

Reply:
ah I know why.. it only converts the data if method is explicitly set like  `{ method: 'GET', data }`. It should probably be changed to look for something like `if (method && method. !== 'GET') options.body = options.data;`

Anyhow there's a separate card for this pattern refactor where we can do it.

Reply:
Hmm makes sense. So leave as is for now?